File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    # 두 배열의 길이 비교
    if len(arr1) > len(arr2):
        return 1
    elif len(arr1) < len(arr2):
        return -1
    
    # 길이가 같을 경우, 원소의 합 비교
    sum1 = sum(arr1)
    sum2 = sum(arr2)
    
    if sum1 > sum2:
        return 1
    elif sum1 < sum2:
        return -1
    else:
        return 0

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.js
function solution(arr1, arr2) {
    var sum1 = 0;
    var sum2 = 0;
    
    if(arr1.length > arr2.length) {
        return 1
    } else if(arr1.length < arr2.length){
        return -1;
    } else {
        for(let x of arr1) {
            sum1 += x;
        }
        for(let y of arr2) {
            sum2 += y;
        }
        if(sum1>sum2) {
            return 1;
        }else if(sum1<sum2) {
            return -1;
        }else {
            return 0;
        }
    }
}